portfolio/src/main/webapp/script.js

```diff
@@ -85,8 +85,13 @@ function goBack() {
 /**
  * Fetches the response of "/data".
  */
-function loadAndShowData() {
-    fetch("/comment").then(response => response.json()).then((json) => {
+function loadAndShowComments() {
+    let maxNumberOfComments = document.getElementById("quantity").value;
+    if (maxNumberOfComments.length == 0) {
```

Comment:
Is "maxNumberOfComments" a String or int?

Reply:
it is a string

Reply:
but you're assign it = 0?